bin/smoke

```diff
@@ -216,8 +224,7 @@ end
 
 begin
   @options, @args = parse_options
-  application = @args[0]
-  validate_application(application)
+  application = @args[0].split(/ /) # quick hack, does not consider blanks in file names
```

Comment:
OK, I had a couple of ideas.

  1. Write a three-line function that does the job, with tests.
  2. Require people to write a batch script wrapping the application on Windows.
  3. Instead of providing the application on the command line, have a `command` file with one "thing" per line, just like *.args* files.